src/styles/blocks/card.scss

```diff
@@ -0,0 +1,115 @@
+.box {
+  display: grid;
+  grid-template-columns: repeat(4, auto);
+  justify-content: center;
+
+  margin-inline: auto;
+}
+
+.card {
+  box-sizing: border-box;
```

Comment:
move to `*` selector and delete here and  https://github.com/mate-academy/layout_catalog/pull/3617/files#diff-efc436905cb807e5305d392062b3125e8ef099b2665dfbb262301b5e1355a96eR93